bonfire/bonfire.py

```diff
@@ -903,65 +914,115 @@ def _cmd_process(
     print(json.dumps(processed_templates, indent=2))
 
 
-def _get_namespace(requested_ns_name, name, requester, duration, pool, timeout, local, force):
+def _get_namespace(
+    requested_ns_name,
+    name,
+    requester,
+    duration,
+    pool,
+    timeout,
+    local,
+    force,
+    using_current=False
+):
     reserved_new_ns = False
 
     if not has_ns_operator():
         if requested_ns_name:
             ns = Namespace(name=requested_ns_name)
+            return ns.name, requested_ns_name
```

Comment:
```suggestion
            return ns.name, False
```

The second value of the tuple should be 'True' or 'False' -- it indicates whether a namespace was newly reserved by bonfire when `_get_namespace` was invoked. In this area of the code, if there's no namespace operator, it is always 'False'

Reply:
It is false from ` reserved_new_ns = False` (line 928). Do you want this to be explicit here?

Reply:
Disregard, there is a wrong var used. Fixed.